simulator/risc_v/riscv_instr.cpp

```diff
@@ -232,7 +338,7 @@ template<typename T>
 RISCVInstr<T>::RISCVInstr( uint32 bytes, Addr PC)
     : BaseInstruction<T, RISCVRegister>( PC, PC + 4), instr( bytes)
 {
-    const auto& entry = find_entry<MyDatapath>( bytes);
+    const auto& entry = find_entry<MyDatapath>( bytes, sizeof( T));
```

Comment:
We do not use `sizeof` to get bit width of the number (e.g. `sizeof(uint128) != 16` for Boost library). Instead, we have a macro `bitwidth<uint128>`